skbeam/core/fitting/lineshapes.py

```diff
@@ -486,7 +486,7 @@ def poisson_dist(bin_values, K):
     the ``Notes`` section of the ``nbinom_dist()`` docstring
 
     .. math::
-        P(K) = \\frac{<K>^K}{K!}\exp(-<K>)
+        P(K) = \\frac{<K>^K}{K!}\\exp(-<K>)
```

Comment:
I think those are two valid ways to write math equations, which should be rendered into images. I don't think that the rendering will work in the current form (`\` should be used instead of `\\`). Leaving `\` causes both ``pytest`` and ``flake8`` to complain and someone started changing '\'s to '\\' long time ago, I had to do it only in a few places. I think that putting documentation in order and making sure that all LaTeX equations are rendered correctly is a separate job. But even in the current state they are readable.

Reply:
We can also make the whole docstring a 'raw' string and remove the double `\\` everwhere.   

Python uses `\` as an escape charter (ex`'\n'` -> new line, `'\\n'` -> `['\', 'n']`).  It used to be that you only had to escape the `'\'` before letters that formed valid escape characters (but _could_ anywhere).  In newer versions of python they made a single `'\'` before a non-escape code character a warning and iirc is an error on cpython master branch.

Reply:
All sounds good, but for a separate PR :)

Reply:
I've seen that ``numpy`` docstrings are all 'raw' stings even if they don't have embedded equations. I think that I will make a separate PR for docs.